guides/v2.1/coding-standards/docblock-standard-general.md

```diff
@@ -364,7 +368,7 @@ Functions and methods must have:
     For example: `@return Config|null`.
     The DockBlock needs to explain what situations return `null`.
 
-    Another example: `@param FileInterface | null`.
+    Another example: `@param FileInterface|null`.
```

Comment:
Any reason you didn't opt for the nullable operator? `@param ?FileInterface`

Reply:
I find it strange to use nullable on `@param` annotations. Is that a good idea? I honestly don't know. So far I've never encountered that in annotations. Happy to change it if you think it's good and PHPStorm understands it.